Packet++/header/ArpLayer.h

```diff
@@ -46,8 +48,8 @@ namespace pcpp
 	 */
 	enum ArpOpcode
 	{
-		ARP_REQUEST = 0x0001, ///< ARP request
-		ARP_REPLY   = 0x0002  ///< ARP reply (response)
+		ARP_REQUEST = 0x0001,  ///< ARP request
+		ARP_REPLY = 0x0002     ///< ARP reply (response)
```

Comment:
I think we need to move the comments on top of every enum value, like we usually do in enums

Reply:
I think this is the only doxygen coding style no for enum no ? 

https://stackoverflow.com/questions/13752191/documenting-enum-values-with-doxygen

Reply:
@seladb I think this is not in the scope of this PR. Feel free to open new issues for non-format-related issues.

Reply:
@clementperon we document most of our enums in a different way, for example: https://github.com/seladb/PcapPlusPlus/blob/d21c05ccf8cd039a8faef77de02e2f7d7e1549b4/Packet%2B%2B/header/DnsLayerEnums.h#L12

Reply:
@tigercosmos I agree it doesn't have to be in the scope of this PR. I'll let you decide if you want to fix it or not